docs/2.guide/2.directory-structure/2.env.md

```diff
@@ -33,11 +33,31 @@ npx nuxi dev --dotenv .env.local
 
 When updating `.env` in development mode, the Nuxt instance is automatically restarted to apply new values to the `process.env`.
 
-## Production Preview
+## Production
 
 **After your server is built**, you are responsible for setting environment variables when you run the server.
 
-Your `.env` file will not be read at this point. How you do this is different for every environment.
+Your `.env` files will not be read at this point. How you do this is different for every environment.
+
+This design decision was made to ensure compatibility across various deployment environments, some of which may not have a traditional file system available, such as serverless platforms or edge networks like Cloudflare Workers.
+
+Since `.env` files are not used in production, you must explicitly set environment variables using the tools and methods provided by your hosting environment. Here are some common approaches:
+
+* Pass the environment variables as arguments using the terminal
+
+```bash [Terminal]
+DATABASE_HOST=mydatabaseconnectionstring node .output/server/index.mjs
+```
```

Comment:
This (and the other sections can probably be indented three spaces so they are nested under the list items

Reply:
Indented the sections, pushed changes.